src/scss/components/_label.scss

```diff
@@ -11,7 +11,7 @@
   &.field-required:after {
     content: '*';
     color: var(--of-color-danger);
-    z-index: 1; // otherwise this is above the progress indicator
+    z-index: 0; // otherwise this is above the progress indicator
```

Comment:
both seem to work visually - what is/was the problem with the z-index being `1`?

Reply:
Amazingly this differs in Firefox and Chrome on Ubuntu: in FF the asterisks for required fields remain visible when scrolling through the progress indicator mobile view. I think I will need to test this on PC and Mac before making a decision.

Reply:
Indeed testing FF on Mac and Windows did not reproduce the same bug - so I'm switching to Chrome for front-end.